package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "fs-plus": "2.x",
-    "temp": "~0.6.0"
+    "temp": "~0.6.0",
+    "underscore": "*"
```

Comment:
Probably not a good idea to just leave this wide open. Atom uses underscore-plus `^1.6.6` currently:

https://github.com/atom/atom/blob/master/package.json#L69